content/en/blog/_posts/2025-02-21-introducing-gateway-api-inference-extension/index.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: blog
+title: "Introducing Gateway API Inference Extension"
+date: 2025-02-20
+slug: introducing-gateway-api-inference-extension
+draft: true
+---
+
+**Authors**: Abdullah Gharaibeh (Google), Kellen Swain (Google), Daneyon Hansen (Solo.io), Jiaxin Shan (Bytedance)
```

Comment:
Hi. We now list the authors within [front matter](https://gohugo.io/content-management/front-matter/); please edit accordingly.

Reply:
Resolved by commit 3b5f77a.